docs/quickstart.md

```diff
@@ -0,0 +1,44 @@
+# Quick Start
+
+Fork and clone
+```bash
+mkdir -p $GOPATH/src/sigs.k8s.io
+cd $GOPATH/src/sigs.k8s.io
+
+# fork https://github.com/kubernetes-sigs/application
+# clone
+GITHUBID=<githubid>
+git clone git@github.com:${GITHUBID}/application.git $GOPATH/src/sigs.k8s.io/kubebuilder
+```
+
+Run locally
+```bash
+# create local cluster
+make e2e-setup
```

Comment:
Did you forget to check in the `e2e-setup` and `e2e-cleanup` goals? I couldn't find them in the Makefile. @barney-s

Reply:
its in the other CL pending merge #159